docs/plans/search-analytics.md

```
@@ -0,0 +1,76 @@
+
+### 4. `search_modal_open` Event
+
+Fires when search modal is opened.
+
+### 5. `search_modal_close` Event
+
+Fires when search modal is closed.
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)